docs/preview/02-Features/02-message-handling/04-event-hubs-azure-functions.md

```diff
@@ -0,0 +1,242 @@
+﻿---
+title: "Azure EventHubs message handling for Azure Functions"
+layout: default
+---
+
+This article describes how you can use Arcus' message handler concept with Azure Functions; allowing you to more easily port your business logic from/to Azure Functions.
+
+# Azure EventHubs message handling for Azure Functions
+While our default message pump system provides a way to receive, route, and handle incoming EventHubs messages which are, unfortunately, not supported in Azure Functions.
+Today, Azure Functions acts as a message receiver meaning that the function is triggered when a message is available but does not handle message routing and handling. However, in this case, it acts as the message pump.
+
+Following terms are used:
+- **Message handler**: implementation that processes the received message from an Azure EventHubs subscription. Message handlers are created by implementing the `IAzureEventHubsMessageHandler<TMessage>`. This message handler will be called upon when a message is available on the Azure EventHubs subscription.
+- **Message router**: implementation that delegates the received Azure EventHubs message to the correct message handler.
+
+That's why we extracted our message routing functionality so you can call it directly from your Azure Function.
```

Comment:
That's why refers to a reason, but I cannot find the reason ?

Reply:
Removed sentence as it wasn't bringing any added-value. 👍